### ISSUE TITLE
Program error

### ISSUE THREAD

**Issue by JusticeSword** [morris/vstools#8]
So if i use the _"Run in browser"_ link provided on this git, it works just fine, however attempting to launch from the downloaded code (Via the "**Index**" chrome html link) brings me a similar screen as the linked browser version, albeit with a completely black background and will not load the model files.

Not a big deal as i can simply bookmark this page but would be nice if i could get the working offline version.

Thanks for your hard work on this.

**Comment by morris** [comment]
yeah, opening the local index.html via file:// protocol will not work because importing ES6 modules is not allowed in that case. you'd need to run a webserver locally. i've included one (`npm install` once and then `npm run serve`) should work. or something like `php -S localhost:8080` if you have php. cheers!

**Comment by JusticeSword** [comment]
Im afraid most of my ability lies in the artistic side and not the code elements, however i am already finding great use for your program even in browser, so we may consider this issue closed if you like, thanks again!

**Comment by morris** [comment]
sorry if that came out a bit technical :) i'll update the readme for offline usage. thanks for your interest, really appreciated!
